games-puzzle/pathological/pathological-1.1.3.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="An enriched clone of the game Logical"
+DESCRIPTION="Pathological is an enriched clone of the game \
+"Logical" by Rainbow Arts. \
+To solve a level, fill each wheel with four marbles of matching color. \
+Various board elements such as teleporters, switches, filters, etc., \
+make the game interesting and challenging. \
+New levels can be created using your favorite text editor.clea"
```

Comment:
Suggestion:
```diff
-SUMMARY="An enriched clone of the game Logical"
-DESCRIPTION="Pathological is an enriched clone of the game \
-"Logical" by Rainbow Arts. \
-To solve a level, fill each wheel with four marbles of matching color. \
-Various board elements such as teleporters, switches, filters, etc., \
-make the game interesting and challenging. \
-New levels can be created using your favorite text editor.clea"
+SUMMARY="A puzzle game consisting of marbles that roll along paths"
+DESCRIPTION="Pathological is an enriched clone of the game Logical by Rainbow \
+Arts.
+To solve a level, fill each wheel with four marbles of matching color.
+Various board elements such as teleporters, switches, filters, etc., make the \
+game interesting and challenging.
+New levels can be created using your favorite text editor."
```

Reply:
Keeping the 80 characters restriction and create new lines (without "\" at the end)